source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -0,0 +1,256 @@
+
+.. warning::
+
+   The reader is assumed to have basic knowledge of what a blockchain and smart
+   contract is, and some experience with Rust_.
+
+
+Preparation
+===========
+
+Before you start, make sure you create an account on your favourite cloud provider to set up your instance.
+
+You are now ready to select an instance on the cloud provider that matches the following requirements to run a testnet node:
+
+.. list-table::
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Hardware (Testnet node)
+     - Recommended
+   * - CPU (Core)
+     - ?
+   * - RAM (Memory)
+     - 8 GB
+   * - Storage
+     - 50 GB
+   * - Operating system
+     - Ubuntu 20.04 x64
+
+.. Note::
+   The above requirements are simple and meant for a testnet node only. Please have a look at the :ref:`requirements <system-requirements-node-mainnet>` for mainnet nodes.
+
+Syncing a testnet node
+======================
+
+The :ref:`run a node <node-requirements>` guide will help you set up your instance correctly. It is recommended to use an ubuntu instance on a server in the cloud for this tutorial.
+
+.. Note::
+   It is technically fine to run your testnet node locally on your machine instead of on a server in the cloud. Since blockchain nodes have to run 24/7 to be up-to-date with the blockchain, you have to run your local machine 24/7. Alternatively, if you don't want to run your local machine 24/7, you can let your node catch up whenever you start your machine. Because this takes some time, this tutorial recommends a cloud provider setup for convenience.
+
+Start the syncing process of the testnet node by following the guide for your platform :ref:`Ubuntu<ubuntu-node>`, :ref:`Docker<docker-node>`, :ref:`Windows<windows-node>`, or :ref:`MacOS<macos-node>`.
+
+You should find your node name on the `Concordium testnet dashboard <https://dashboard.testnet.concordium.com/>`__. It will take less than a day until your testnet node is fully synced. You can observe the syncing process by watching the finalization length of your node. Wait until the ``Fin Length`` (finalization length) of your node is the same as the highest value used by the majority of nodes.
+
+.. Note::
+   To allow the network dashboard to display nodes in the network and their current status, a node must run a collector. The collector is enabled by the default start-up script but can be disabled.
+
+.. image:: ./images/pb_tutorial_13.png
+   :width: 100 %
+
+Alternatively, you can query the syncing state of your node with the ``concordium-client`` tool. You will learn how to set up the ``concordium-client`` tool in the next section. Feel free to come back to this command after reading the next section.
+
+.. code-block:: console
+
+   ./concordium-client consensus status --grpc-port 10001
```

Comment:
I would add the specific value they are looking for, namely, `Last finalized block height` 😊